### PR TITLE
Worked on UX for edit page (v2)

### PR DESCRIPTION
**Pull request by Squishibutz**
* added water.css -> auto (light&dark mode possible)
* fixed add button (lost data on click)
* added feedback to save
* added pile url and copy button
* added back to pile link

image

### CODE REVIEW

**Comment by itsmethemojo** [comment]
i see 2 improvements to this gorgeous frontend makeover

1. add a cache invalidation parameter on the css includes like `?d=20210524` to make sure they will be loaded. later this should be done by an intelligent build process
2. document where that minified auto.css came from, maybe that can be later also pulled from somewhere else within the build process

**Comment by Squishibutz** [comment]
> i see 2 improvements to this gorgeous frontend makeover
> 
> 1. add a cache invalidation parameter on the css includes like `?d=20210524` to make sure they will be loaded. later this should be done by an intelligent build process
> 2. document where that minified auto.css came from, maybe that can be later also pulled from somewhere else within the build process

updated my PR

File: public/create.html [hunk 1]
```diff
@@ -5,7 +5,8 @@
   <head>
   	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   	<title>create a new pile</title>
-    <link rel="stylesheet" href="/style.css">
+    <link href="/auto.css?d=20210524" type="text/css" rel="stylesheet">
+    <link href="/custom.css" type="text/css" rel="stylesheet">
```
Comment (itsmethemojo) [inline public/create.html]: one last thing. i think you missed  the cachebreaker on the custom.css (in multiple files)

Comment (Squishibutz) [reply public/create.html]: added

**Comment by itsmethemojo** [comment]
LGTM. Deploy will start automaticly after merge

**Pull request merged**